src/external/visualization_integration.cuh

```diff
@@ -52,7 +53,8 @@ __host__ std::string vector_double_to_json_array(const std::vector<double> &v);
  *      duplication (because there are two main functions for cpp and cuda), which might mean for you that you don't
  *      want to use this function but want to write your own request sender
  */
-__host__ bool send_particles_to_visualization(const std::string &url, MapNode *nodes, int n_of_nodes);
+__host__ bool send_particles_to_visualization(const std::pair<std::string, std::string> &urls, MapNode *nodes, int n_of_nodes,
+                                              Polyhedron *polyhedron, int n_of_faces);
```

Comment:
Do you think one function should send both the polyhedron and the data about particles each time we want to communicate with the visualization module? I doubt that, but if so, you should rename the function and update the docs accordingly